src/coronawhy_vt/utils/local_processing.py

```diff
@@ -0,0 +1,375 @@
+# -*- coding: utf-8 -*-
+
+"""Temporary storage for utils used in the processing of local data structures."""
+# TODO function names are in camelcase suggest refactoring to conform to PEP-8
+import json
+import os
+from typing import Callable
+
+import dateutil.parser as parser
+import numpy as np
+import pandas as pd
+import pylab
+import spacy
+
+# TODO All of these functions open a file and do not close it, putting them in the same class would solve some issues
+# TODO lowercase?
+
+
+def make_ngram_map_and_list(filename: str, min_word_len: int = 6, min_num_tokens: int = 0,
+                            max_num_tokens: int = 4) -> (list, dict):
+    """Given the path of a file of tokens, replace every " " with "_".
+
+    :param filename: The path of the file
+    :param min_word_length: Minimum length of vocab entry to be accepted
+    :param min_num_tokens: Minimum number of tokens in the vocab entry
+    :param max_num_tokens: Maximum number of tokens in the vocab entry
+
+    :return (names, ngram_map): (Original tokens with spaces instead of underscores
+        {names: names_with_space_repaced_by_underscores})
+    """
+    ngram_map = {}
+    names = []
+    all_names = []
+
+    file = open(filename).readlines()
+    for line in file:
```

Comment:
iterating over lines in a file is good enough - you don't have to do file.readlines()